codex/stores/memorystore.nim

```diff
@@ -0,0 +1,186 @@
+## Nim-Codex
```

Comment:
If there is a compatibility problem with just using `vendor/lrucache` then it should be clearly documented here (or at the use site).  OTOH, if there is no problem, just using it directly may be better.

Reply:
Ok.  Maybe mentioning "Replacement policy may be other than LRU" in a comment is enough, if that is the crux.  Just something for casual readers later.  Just a thought.

Reply:
> could also be used as a third option from the CLI

Is this really a practical use case? Not sure if I see one here.